test/interactiveutil.jl

```diff
@@ -0,0 +1,110 @@
+### Testing download(url,[localfile],[silent=false])
+
+"Runs an expression, and returns stdout, stderr, and the result."
+function capture_output(expression)
```

Comment:
Could avoid using `eval` if you would change the signature take in a function and its arguments:
```julia
function capture_output(func::Function, args...; kwargs...)
    ...
    result = func(args...; kwargs...)
    ...
end

```